RPIPs/rpip-remove_retro_cap.md

```diff
@@ -150,20 +158,13 @@ To ensure a smooth transition and maintain operational continuity, the committee
 
 ###### Committee Member Replacement
 
-- Once the decision to remove the administrator has been made, a backup committee member will be assigned to assume the administrator's responsibilities temporarily during the hiring process.
+- If the GMC Administrator role is vacant at any time, a backup committee member will be assigned to assume the administrator's responsibilities temporarily during the hiring process. The backup will receive compensation equivalent to the administrator's compensation during the time the backup serves.
```

Comment:
I didn't notice this... but "will be assigned" isn't clear. Assigned by who and how? I might suggest making the GMC treasurer serve as the fallback simply to remove ambiguity or add a step that creates a time without anyone in the role.

Reply:
resolving as newer comment exists